cmd/ctr/utils.go

```diff
@@ -79,3 +85,68 @@ func waitContainer(events execution.ContainerService_EventsClient, id string, pi
 		}
 	}
 }
+
+func forwardAllSignals(containers execution.ContainerServiceClient, id string) chan os.Signal {
+	sigc := make(chan os.Signal, 128)
+	catchAll(sigc)
+	go func() {
+	Signals:
+		for s := range sigc {
+			var sig syscall.Signal
+			for sigStr, sigN := range signalMap {
```

Comment:
I think we can skip it, this is because docker's client works on both windows/linux and across systems, i.e. windows client to linux daemon.  We don't have to worry about this with containerd